packages/core/src/data-editor/data-editor.tsx

```diff
@@ -3440,7 +3440,18 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
                 return gridRef.current?.damage(damageList);
             },
             getBounds: (col, row) => {
-                return gridRef.current?.getBounds(col + rowMarkerOffset, row);
+                if (col === undefined && row === undefined && scrollRef.current && canvasRef.current) {
+                    // Return the bounds of the entire scroll area:
+                    const rect = canvasRef.current.getBoundingClientRect()
```

Comment:
Maybe this path should return a default value if canvasRef.current is undefined instead of getting the bounds of the first column?

Reply:
Changed it to return `undefined` in case canvasRef.current or scrollRef.current is undefined.